packages/service-library/src/servicelib/archiving_utils.py

```diff
@@ -355,15 +357,17 @@ async def archive_dir(
     ::raise ArchiveError
     """
     if not progress_bar:
-        progress_bar = ProgressBarData(num_steps=1)
+        progress_bar = ProgressBarData(
+            num_steps=1, description=f"compressing {dir_to_compress.name}"
+        )
 
     async with AsyncExitStack() as stack:
         folder_size_bytes = sum(
             file.stat().st_size
             for file in _iter_files_to_compress(dir_to_compress, exclude_patterns)
         )
         sub_progress = await stack.enter_async_context(
-            progress_bar.sub_progress(folder_size_bytes)
+            progress_bar.sub_progress(folder_size_bytes, description="...")
```

Comment:
ProgressBarData is a backend entity, not meant to be displayed. Whatever someone does with the ProgressReport is not my problem, but the UI or client one. that is why it is not a display name since there is no display.

Reply:
right, so it's a name not an ID. and a name can be whatever, even three dots.

Reply:
discussed